code/game/machinery/computer/arcade.dm

```diff
@@ -336,10 +336,17 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	enemy_name = "Cuban Pete"
 	name = "Outbomb Cuban Pete"
 
-
 	updateUsrDialog()
 
-
+/obj/machinery/computer/arcade/battle/on_deconstruction()
+	. = ..()
+	if((obj_flags & EMAGGED))
+		if(ismob(usr))
+			to_chat(usr, "<span class='userdanger'>Trying to cheat me? Chick-chicky-boom, chick-chicky boom!</span>")
+			usr.gib()
+		else
+			var/turf/T = get_turf(src)
+			explosion(T, 2, 4, 8, flame_range = 16) //in case it gets destroyed by other means
```

Comment:
make this an explosion that doesn't create a breach but damages user

Reply:
Hard to do since the only time this would ever come up was if someone was throwing an object at it to break it or shot it with a gun.